node/src/components/small_network.rs

```diff
@@ -740,14 +762,17 @@ where
         for request in requests.into_iter() {
             trace!(%request, "processing dial request");
             match request {
-                DialRequest::Dial { addr, span } => effects.extend(
-                    tasks::connect_outgoing(self.context.clone(), addr)
-                        .instrument(span.clone())
-                        .event(|outgoing| Event::OutgoingConnection {
-                            outgoing: Box::new(outgoing),
-                            span,
-                        }),
-                ),
+                DialRequest::Dial { addr, span } => {
+                    info!(is_syncing = ?self.context.is_syncing, "connecting");
```

Comment:
Let's not commit these debug messages here, they will spam the logs. There is detailed information about connection states already available in the outgoing connection statemachine handling.

Reply:
Comment removed in 5b91f0f74616a4c60822d297c58f08523917ec38